app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -1,8 +1,21 @@
-<%= form_with model: @application_form, url: @path, method: @request_method do |f| %>
-  <div class='govuk-form-group'>
-    <%= f.hidden_field @field_name, value: false %>
-    <%= f.govuk_check_box @field_name, true, multiple: false, label: { text: t(checkbox_label) } %>
-  </div>
+<%= form_with model: section_complete_form, url: path, method: request_method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= render(review_component) %>
+
+  <%= f.govuk_radio_buttons_fieldset :completed, classes: 'govuk-!-width-two-thirds ', legend: { text: t('application_form.completed_question'), size: 'm' } do %>
+    <% if hint_text %>
+      <% if hint_text.is_a?(Array) %>
+        <% hint_text.each do |paragraph| %>
+          <div class="govuk-hint"><%= paragraph %></div>
+        <% end %>
+      <% else %>
+        <div class="govuk-hint"><%= hint_text %></div>
```

Comment:
```suggestion
        <p class="govuk-hint"><%= hint_text %></p>
```

Reply:
@paulrobertlloyd - the reason for using `<div>` was because an extra margin was being added by `<p>` - would you rather this be overridden?

![Uploading multi_hint.png…]()

Reply:
@paulrobertlloyd @TobyRet the Design System component uses `<div>`s for hints in radios, for some reason: https://design-system.service.gov.uk/components/radios/

Reply:
@frankieroberto - @paulrobertlloyd has decided to have the multiple hints functionality removed. This will simplify things and allow us to use the form builder component. I'm going to do this in a followup PR

@paulrobertlloyd can you approve the PR pls?